llms/mlx_lm/tuner/utils.py

```diff
@@ -83,15 +84,33 @@ def apply_lora_layers(model: nn.Module, adapter_file: str) -> nn.Module:
     if not os.path.exists(adapter_file):
         raise FileNotFoundError(f"The adapter file does not exist: {adapter_file}")
 
-    adapters = list(mx.load(adapter_file).items())
+    weights, metadata = mx.load(adapter_file, return_metadata=True)
```

Comment:
I'm not very sure if loading the np file with `return_metadata=True` will throw an error or just return None for the metadata.